packages/form-js-editor/src/FormEditor.js

```diff
@@ -103,6 +111,8 @@ export default class FormEditor {
             schema
           });
 
+          this._emit('import.done', { warnings });
```

Comment:
We do fire `import.done` no matter what, i.e. on error, too. Please implement (and test it) in form-js accordingly, too.

Reply:
Done. 👍🏻